src/style.css

```diff
@@ -0,0 +1,45 @@
+* {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 96px;
+  height: 96px;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
```

Comment:
Delete this style, you should only add correct margin to each star

Reply:
Fixed